src/pages/index.tsx

```diff
@@ -85,30 +92,30 @@ const HomePage = () => (
             </Heading>
             <List fontSize="" color="secondaryTextLightmode" spacing="4">
               <ListItem mt="-2">
-                <ListIcon as={FaCheck} color="sky.400" />
+                <ListIcon as={HiOutlineSwitchHorizontal} color="sky.400" />
```

Comment:
```suggestion
                <ListIcon as={HiOutlineSwitchHorizontal} color="sky.700" />
```

The Figma design uses `Sky700` for the icons themselves, they will have a round padding around them with a lighter color, so that the icons become bigger visually.

Might as well use the correct icon color right now. 😄